Meden-Agan-27/travel-worms
# added the index page and the show page for books

What I did: built the index and show page, added a feature that checks which books you can see depending on your preferred language. Made it so the index is available for view when you are not signed in.

How to test:
- Click the link to see all books on the home page on localhost:3000
- Check the list of books, you can see the details page but have to be logged in.
- log in as rafiki@books.com with "password" as password. Check back on the index that you can only see Harold et Maude
- log out
- log in as sano@books.com with "password" as password. Check on the index that you can see all books except Harold et Maude.
